TaskWithStrlen/TaskWithStrlen.cpp

```diff
@@ -0,0 +1,34 @@
+﻿#include <iostream>
+
+int strLength(char str[]) {
+	int count = 0;
+	while (str[count] != '\0')
+	{
+		count++;
+	}
+
+	return count;
+}
+
+int main()
+{
+	char str[] = "Hello World";
+
+	std::cout << strLength(str) << std::endl;
+
+	std::cout << strlen(str) << std::endl;
+
+	for (int i = 0; i < sizeof(str); i++)
+	{
+		if (str[i] == '\0')
+		{
+			str[i] = 'd';
+		}
+	}
+
+	std::cout << strLength(str) << std::endl;
```

Comment:
Please leave a comment that we expect undefined behavior here.

Reply:
Comment was written